### ISSUE TITLE
Invalid default SQS queue attributes

### ISSUE THREAD

**Issue by thomasdc** [localstack/localstack#2857]
# Type of request

bug report

# Detailed description

The default attributes of an SQS queue in localstack version 0.11.3 have changed from previous versions.

These 2 attributes are now added by default:
- "Policy": "None"
- "RedrivePolicy": "None"

The [AWS documentation](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/sqs/get-queue-attributes.html#options) mentions that `RedrivePolicy` contains a JSON value. "None" clearly isn't valid JSON. It doesn't however explicitly mentions `Policy` should be a JSON value as well, but anyhow, the library I'm using (MassTransit) clearly expects it to be JSON. See this [bug report](https://github.com/MassTransit/MassTransit/issues/2023).

## Expected behavior

Previous versions of localstack (up to 0.11.2) simply didn't add the `Policy` and `RedrivePolicy` attributes by default.

# Steps to reproduce

## localstack behavior >= v0.11.3
```
λ aws --endpoint-url=http://localhost:4576 sqs create-queue --queue-name Foobar1             
{                                                           
    "QueueUrl": "http://localhost:4576/000000000000/Foobar1"
}

λ aws --endpoint-url=http://localhost:4576 sqs get-queue-attributes --queue-url http://localhost:4576/queue/Foobar1 --attribute-names All
{
    "Attributes": {
        "ApproximateNumberOfMessages": "0",
        "ApproximateNumberOfMessagesDelayed": "0",
        "ApproximateNumberOfMessagesNotVisible": "0",
        "CreatedTimestamp": "1597967156.664395",
        "DelaySeconds": "0",
        "LastModifiedTimestamp": "1597967156.664395",
        "MaximumMessageSize": "262144",
        "MessageRetentionPeriod": "345600",
        "QueueArn": "arn:aws:sqs:eu-west-1:000000000000:Foobar1",
        "Policy": "None", <----- Not expected
        "RedrivePolicy": "None", <----- Not expected
        "ReceiveMessageWaitTimeSeconds": "0",
        "VisibilityTimeout": "30"
    }
}
```

## localstack behavior < v0.11.3
```
λ aws --endpoint-url=http://localhost:4576 sqs create-queue --queue-name Foobar2                 
{                                                               
    "QueueUrl": "http://localhost:4576/queue/Foobar2"           
}                                                               

λ aws --endpoint-url=http://localhost:4576 sqs get-queue-attributes --queue-url http://localhost:4576/queue/Foobar2 --attribute-names All
{
    "Attributes": {
        "VisibilityTimeout": "30",
        "DelaySeconds": "0",
        "ReceiveMessageWaitTimeSeconds": "0",
        "ApproximateNumberOfMessages": "0",
        "ApproximateNumberOfMessagesNotVisible": "0",
        "ApproximateNumberOfMessagesDelayed": "0",
        "CreatedTimestamp": "1597967269",
        "LastModifiedTimestamp": "1597967269",
        "QueueArn": "arn:aws:sqs:us-east-1:000000000000:Foobar2"
    }
}
```

Was this change in behavior intended?

**Comment by whummer** [comment]
Thanks for reporting @thomasdc . This should be fixed upstream in spulec/moto#3255 and is now available in LocalStack. Can you please give it another try with the `latest` version? Please report here if the problem persists. Thanks

**Comment by thomasdc** [comment]
Yep, the default `Policy` and `RedrivePolicy` attributes are gone in the latest version. Thanks for the fix!
